typegate/src/typegraph/utils.ts

```diff
@@ -11,6 +11,19 @@ export function treeView(tg: TypeGraphDS, rootIdx = 0, depth = 4) {
     const edge = cyan(`${path.edges[path.edges.length - 1] ?? "[root]"}`);
     const idxStr = green(`${idx}`);
     console.log(`${indent}${edge} → ${idxStr} ${type.type}:${type.title}`);
+    if (type.type == "union") {
+      console.log(`${indent}anyOf → ids [${type.anyOf.join(", ")}]`);
```

Comment:
do not forget to use the logger with correct level if you want to keep the logs or remove the console log statement before the merge